styles/styles.css

```diff
@@ -15,35 +15,653 @@ li {
 body {
   background: var(--primary);
   overflow-x: hidden;
   font-family: "Montserrat", sans-serif;
 }
 
-/* fonts */
-h1,
-h2,
-h3,
-h4 {
-  font-weight: normal;
-  line-height: 1.4em;
+.button {
+  display: inline-block;
+  background: #55b8f0;
+  color: var(--primary);
+  border-radius: 30px;
+  padding: 12px 0;
+  font-weight: 500;
+  font-size: 0.8em;
+  line-height: 12px;
+  text-align: center;
+  width: 216px;
 }
-p,
-a,
-li {
-  color: #9893d8;
-  line-height: 1.4em;
-  font-size: 1em;
-}
-h1,
-h3 {
-  font-size: 1.2em;
+
+.button:hover {
+  filter: brightness(95%);
 }
-h2 {
+
+.block-title {
   font-size: 1.6em;
+  font-weight: 600;
+  color: var(--primary-font-color);
 }
-h4 {
+
+.block-small-title {
   font-size: 1.1em;
-}
\ No newline at end of file
+  font-weight: 500;
+  color: var(--secondary-font-color);
+}
+
+.block-description {
+  font-size: 1em;
+  font-weight: 400;
+  color: var(--primary-font-color);
+}
+
+.container {
+  margin: 60px 15%;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 10px;
+  box-sizing: border-box;
+}
+
+.container .block-title,
+.block-small-title,
+.block-description {
+  margin: 8px 0;
+}
+
+.container .button {
+  display: block;
+  margin: 0 auto;
+}
+
+select {
+  padding: 0 5px;
+  margin-left: 10px;
+  width: 60px;
+  height: 25px;
+  border: 1px solid #a4a3ad;
+  border-radius: 4px;
+  font-size: inherit;
+  line-height: inherit;
+  z-index: 1;
+  color: #5c5c6c;
+}
+
+.chevron-down {
+  @apply --chevron-mixin;
+  position: relative;
+}
+.chevron-down::after {
+  content: "";
+  display: block;
+  box-sizing: border-box;
+  position: absolute;
+  width: 10px;
+  height: 10px;
+  border-bottom: 2px solid;
+  border-right: 2px solid;
+  transform: rotate(45deg);
+  left: 4px;
+  top: 2px;
+}
+
+.chevron-left {
+  @apply --chevron-mixin;
+  position: relative;
+}
+.chevron-left::after {
+  content: "";
+  display: block;
+  box-sizing: border-box;
+  position: absolute;
+  width: 10px;
+  height: 10px;
+  border-bottom: 2px solid;
+  border-right: 2px solid;
+  transform: rotate(135deg);
+  left: 4px;
+  top: 2px;
+}
+
+.chevron-right {
+  @apply --chevron-mixin;
+  position: relative;
+}
+.chevron-right::after {
+  content: "";
+  display: block;
+  box-sizing: border-box;
+  position: absolute;
+  width: 10px;
+  height: 10px;
+  border-bottom: 2px solid;
+  border-right: 2px solid;
+  transform: rotate(-45deg);
+  left: 4px;
+  top: 2px;
+}
+/* ========================== End Main styles ========================== */
+
+/* ========================== Navigation ========================== */
+.site-nav {
+  overflow: hidden;
+  background-color: var(--primary);
+}
+
+.toggle,
+[id^="drop"] {
+  display: none;
+}
+
+nav {
+  margin: 0;
+  padding: 0;
+  background-color: var(--primary);
+  padding: 0 15%;
+}
+
+#logo {
+  display: flex;
+  align-items: center;
+  margin-left: 0;
+  padding-left: 0;
+  float: left;
+  margin: 30px 0;
+}
+
+#logo span {
```

Comment:
Avoid applying styles to HTML tags.
These styles were also applied to mobile toggler.